### PR TITLE
Fix html code in locale files (see #6859)

### PR DESCRIPTION
**Pull request by ghost**
Fixed some html code discussed in issue #6859 and some other x/html inconsistencies.

### CODE REVIEW

**Comment by keysmashes** [comment]
I would suggest that we continue to use `<br>` instead of `<br/>` or `<br />`, because that is the style used throughout the codebase for the most part*, and it makes no difference to browsers.

https://stackoverflow.com/questions/1946426/html-5-is-it-br-br-or-br

*searching for `<br>` in the Tasks page gave 220 results, `<br/>` gave 11 and `<br />` gave 3

**Comment by Alys** [comment]
The tasks page isn't a good indicator of the codebase because most of those br tags will be automatically generated by the markdown parser or templating engine or other things.

If you do a git grep for br ignoring the locales files, you find:

```
<br/> 8 lines
<br /> 4 lines
<br> 2 lines
```

Those numbers are too small to indicate a convention for any of them.

In the locales/en directory (not including this PR's changes) you find:

```
<br/> 6 lines
<br /> 0 lines
<br> 34 lines
```

This PR probably changes all of those to `<br />`; it looks thorough at a glance.

Normally, my personal preference is for `<br />` because having valid XHTML doesn't hurt and can be useful. You could make a case for leaving them as `<br>` for the sake of not making extra work now for the translators. However if we find that later we'd benefit from `<br />` being consistently used, then changing later would be more work than changing now. It's not inconceivable that we'd want XHTML later so that we could do automatic processing/conversion of the text. I think I'm in favour of moving to `<br />` now.

Other opinions?

**Comment by crookedneighbor** [comment]
I'd prefer `<br>` as it is what the jade templating language renders `br` as and is valid HTML5 (current standard over XHTML).

**Comment by keysmashes** [comment]
OTOH, all three are valid HTML5.

> Start tags consist of the following parts, in exactly the following order:
> 1. A "<" character.
> 2. The element's tag name.
> 3. Optionally, one or more attributes, each of which must be preceded by one or more space characters.
> 4. **Optionally, one or more space characters.**
> 5. **Optionally, a "/" character, which may be present only if the element is a void element.**
> 6. A ">" character.

https://www.w3.org/TR/html-markup/syntax.html#syntax-start-tags

**Comment by ghost** [comment]
My tendency was obviously to go toward valid xhtml, but I can change back to `<br>` if we can agree on any convention. As far as I'm concerned, @crookedneighbor made a strong point with jade. You just tell me.

**Comment by ghost** [comment]
I've reverted to `<br>` as it seems to be the best idea. All that's left is the html corrections on the few files that were explicitly mentioned on the original issue.

File: common/locales/en/questsContent.json [hunk 1]
```diff
@@ -69,7 +69,7 @@
...
+  "questVice1Notes": "They say there lies a terrible evil in the caverns of Mt. Habitica. A monster whose presence twists the wills of the strong heroes of the land, turning them towards bad habits and laziness! The beast is a grand dragon of immense power and comprised of the shadows themselves: Vice, the treacherous Shadow Wyrm. Brave Habiteers, stand up and defeat this foul beast once and for all, but only if you believe you can stand against its immense power. <br><h3>Vice Part 1: </h3><br>How can you expect to fight the beast if it already has control over you? Don't fall victim to laziness and vice! Work hard to fight against the dragon's dark influence and dispel his hold on you!",
```
Comment (ghost) [inline common/locales/en/questsContent.json]: Have you checked that there would be no visual difference between
`<p>text</p>` and just `text`
for the strings in this file?

Comment (Alys) [inline common/locales/en/questsContent.json]: @Blablux When paragraphs and headings are mixed, `<p>` tags should be used. 

This is better HTML:
`<p>... power. </p><h3>Vice Part 1: </h3><p>How ... </p>`

than this:
`... <br><h3>Vice Part 1: </h3><br>How ...`

Comment (ghost) [inline common/locales/en/questsContent.json]: I reverted to `<p>` for this part. But this leads me to another question. _Vice Part 1_ is the only quest using a heading tag. I did not find another one on any part of the _Vice_ quests or any other quest. While it is certainly not a problem, this seems odd to me. Should I open an issue about it to discuss what we can do ? Or should I just leave it like that as it is not a real problem ?

**Comment by crookedneighbor** [comment]
Squashed your commits and merged them in. 

This will count towards a contributor tier! What's your User ID? You can find it in [Settings > API](https://habitrpg.com/#/options/settings/api). Be sure to only copy the User ID and _not_ your API Token.

**Comment by ghost** [comment]
Wow, I did not thought it was worth a contributor tier ! So, well, thank you !
My User ID is a75eb6f8-5a29-482f-a971-7aff7438da86

**Comment by crookedneighbor** [comment]
I've notated this toward your next tier and added Blacksmith to your title.

**Pull request closed without merging**